### PR TITLE
Fix: Support Python 3

### PR DESCRIPTION
**Pull request by steils**

### CODE REVIEW

**Comment by matthijskooijman** [comment]
Also: For the Debian package, I would be interested in having this fixed in a release. Debian is planning to drop support for Python2, so I'd rather switch to Python3 soon to support the Python2-removal process.

File: scripts/md5list.py [hunk 1]
```diff
@@ -33,9 +33,9 @@
 			md5call = ["md5", "-r"]
 		else:
 			md5call = ["md5sum"]
-		md5call = md5call + ["src/"+string.strip(separate[1])]
+		md5call = md5call + ["src/"+separate[1].strip()]
 		md5sum = subprocess.Popen(md5call, stdout=subprocess.PIPE).communicate()[0]
-		md5sum = string.split(md5sum)
+		md5sum = md5sum.split()
 		res = "%-32s = %s\n" % (separate[1], md5sum[0])
```
Comment (steils) [inline scripts/md5list.py:39]: I am very sorry for this mistake. At this point `b'8d10ea15d5a180873ce4195011fe9f5f'` will be written, and OpenTTD will fail to read it:
```
dbg: [grf] Malformed MD5 checksum specified for: tttheme2.mid (in /usr/share/games/openttd/gm/openmsx-0.3.1/openmsx.obm)
```

`md5sum[0].decode()` is needed.

**Comment by steils** [comment]
Sorry for the mistake I made in the initial PR.
Anyway, @LordAro @planetmaker could you please review and accept it?

**Comment by LordAro** [comment]
Sorry, this repo doesn't (yet) have notifications set up in the same way as the other repos, so changes can get lost.

Why have you replaced indentation with tabs? Should be 4 spaces everywhere, surely?

**Comment by steils** [comment]
@LordAro Didn't know about your coding style. Just saw a mix of tabs and spaces. Changed everything to tabs for consistensy (not 4 spaces because of habit). I'll change everything to 4 spaces, if it is a preferred style, of course.

**Comment by LordAro** [comment]
Not exactly surprising, the rest of OTTD (C/C++) uses tabs, there was some overlap back in the old days.

These days it's PEP8 or GTFO ;)

**Comment by steils** [comment]
Changed tabs to spaces, force-pushed

**Pull request merged**